compiler/ir/backend.wasm/src/org/jetbrains/kotlin/backend/wasm/ir2wasm/WasmCompiledModuleFragment.kt

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.backend.wasm.ir2wasm
+
+import org.jetbrains.kotlin.wasm.ir.*
+import org.jetbrains.kotlin.backend.wasm.lower.WasmSignature
+import org.jetbrains.kotlin.ir.declarations.IrDeclarationWithName
+import org.jetbrains.kotlin.ir.declarations.IrExternalPackageFragment
+import org.jetbrains.kotlin.ir.symbols.*
+import org.jetbrains.kotlin.ir.util.fqNameWhenAvailable
+import org.jetbrains.kotlin.ir.util.getPackageFragment
+
+class WasmCompiledModuleFragment {
+    val functions =
+        ReferencableAndDefinable<IrFunctionSymbol, WasmFunction>()
+    val globals =
+        ReferencableAndDefinable<IrFieldSymbol, WasmGlobal>()
+    val functionTypes =
+        ReferencableAndDefinable<IrFunctionSymbol, WasmFunctionType>()
+    val structTypes =
+        ReferencableAndDefinable<IrClassSymbol, WasmStructDeclaration>()
+    val classIds =
+        ReferencableElements<IrClassSymbol, Int>()
+    val interfaceId =
+        ReferencableElements<IrClassSymbol, Int>()
+    val virtualFunctionId =
+        ReferencableElements<IrFunctionSymbol, Int>()
+    val signatureId =
+        ReferencableElements<WasmSignature, Int>()
+    val stringLiteralId =
+        ReferencableElements<String, Int>()
+
+    val runtimeTypes =
+        ReferencableAndDefinable<IrClassSymbol, WasmGlobal>()
+
+    val classes = mutableListOf<IrClassSymbol>()
+    val interfaces = mutableListOf<IrClassSymbol>()
+    val virtualFunctions = mutableListOf<IrSimpleFunctionSymbol>()
+    val signatures = LinkedHashSet<WasmSignature>()
+    val stringLiterals = mutableListOf<String>()
+
+    val typeInfo =
+        ReferencableAndDefinable<IrClassSymbol, ConstantDataElement>()
+    val exports = mutableListOf<WasmExport<*>>()
+
+    //
```

Comment:
?

Reply:
missed it :( will remove in the next PR.